changelog.d/574.deprecation.rst

```diff
@@ -0,0 +1,11 @@
+The ``cmp`` argument to ``attr.s()`` and ``attr.ib()`` is now deprecated.
+
+Please use ``eq`` to add equality methods (``__eq__`` and ``__ne__``) and ``order`` to add ordering methods (``__lt__``, ``__le__``, ``__gt__``, and ``__ge__``) instead – just like with `dataclasses <https://docs.python.org/3/library/dataclasses.html>`_.
+
+Both are effectively ``True`` by default but it's enough to set ``eq=False`` to disable both at once.
```

Comment:
It seems to not just be "effectively" True, from the below it looks literally True, no?

Reply:
Literally it's `None` and the `True` is derived if `cmp` is kept `None` too. Or am I missing something?

Reply:
Ahhh ok, got it (the "below" I was seeing was the repr output, but yeah I see the actual argument default is as you say).